src/components/AddressResolver/EndpointEntry/EndpointEntry.tsx

```diff
@@ -135,92 +137,136 @@ export const EndpointEntry: FunctionComponent<EndpointEntryProps> = ({
           entityLookup: features.entityLookup?.location,
         },
       });
+
+      showOverlay(<AddResolverConfirmation />);
     } catch (e) {
       onSetAllError(e.message);
     }
     setIsLoading(false);
   };
 
   return (
-    <tr className={className}>
-      <th>
+    <div
+      className={`flex flex-col bg-white rounded-xl shadow-lg w-80 mt-6 pt-12 pb-8 md:even:bg-cerulean-50 ${
+        isEditable ? "h-130" : "h-auto"
+      } md:bg-none md:w-full md:h-auto md:rounded-none md:shadow-none md:mt-0 md:p-4`}
+      onMouseOver={() => setHoverState(true)}
+      onMouseOut={() => setHoverState(false)}
+    >
+      <div className="flex flex-col md:flex-row">
+        <div
+          className={`flex flex-col text-2xl text-cloud-300 mr-3 ${
+            hoverState && !isEditable ? "opacity-1" : "opacity-0"
+          }`}
+        >
+          <i className="fas fa-sort-up hover:text-cloud-900" onClick={onMoveEntryUp} />
+          <i className="fas fa-sort-down -mt-5 hover:text-cloud-900" onClick={onMoveEntryDown} />
```

Comment:
generally avoid negative margin as a means of alignment, unless u sure. in this case it's causing overlapping making it unable to click `up` icon

Reply:
done. changed to altering the line-height between the arrows.